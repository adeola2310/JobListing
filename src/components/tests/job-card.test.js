import React from "react";
import {shallow, mount, render} from 'enzyme';
import JobCard from "../Job-card/Job-card";


it('should render the JobCard component',  () =>{
    // using enzyme shallow
    // expect(shallow(<JobCard/>).length).toEqual(1)

    expect(shallow(<JobCard/>)).toMatchSnapshot();
}); 