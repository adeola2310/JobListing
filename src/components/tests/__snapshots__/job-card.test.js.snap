// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the JobCard component 1`] = `ShallowWrapper {}`;
